translations/de-de.yaml

```diff
@@ -87,6 +88,9 @@ caluma:
       supportsMarkdownPrefix: "Dieses Feld unterstützt"
       markdown: "Markdown"
       dataSource: "Datenquelle"
+      formatValidators: "Validierung"
+      chooseValidation: "Wähle eine Validierung"
```

Comment:
I'd suggest either the formal "Bitte wählen Sie eine Validierung" or porbably even better the already existing "-- bitte wählen --" (`choose`, L103).

Reply:
Done